### PR TITLE
Ensure blazor client side logger checks for enabled log levels

### PR DESCRIPTION
**Pull request by ghidello**
Added a check for IsEnabled in the log method of WebAssemblyConsoleLogger

Addresses #12859

### CODE REVIEW

**Comment by ghidello** [comment]
I didn't find a way to write an unit test so I just added this code 
```
@code{
    protected override void OnInitialized()
    {
        base.OnInitialized();
        Logger.LogDebug("Debug messages should not be logged!");
        Logger.LogWarning("Warning messages should be logged");
    }
}
```
in the Index.razor page of the StandaloneApp project and verified that the Debug log was shown before the change and hidden after it.

File: src/Components/Blazor/Blazor/src/Services/WebAssemblyConsoleLogger.cs [hunk 1]
```diff
@@ -20,6 +20,7 @@ public bool IsEnabled(LogLevel logLevel)
 
         public void Log<TState>(LogLevel logLevel, EventId eventId, TState state, Exception exception, Func<TState, Exception, string> formatter)
         {
+            if (!IsEnabled(logLevel)) return;
```
Comment (mkArtakMSFT) [inline src/Components/Blazor/Blazor/src/Services/WebAssemblyConsoleLogger.cs]:
```suggestion
            if (!IsEnabled(logLevel))
            {
                return;
            }
            
```

Comment (mkArtakMSFT) [reply src/Components/Blazor/Blazor/src/Services/WebAssemblyConsoleLogger.cs]: Can you please apply the proper formatting before we can merge this.

**Comment by SteveSandersonMS** [comment]
Thanks, @ghidello! This is now merged.

**Pull request merged**